### PR TITLE
Implemented PostgreSQL and Environment Variables.

### PR DESCRIPTION
**Pull request by ichiragrathod**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -8,18 +8,21 @@ default: &default
   adapter: sqlite3
```
Comment (poonamsimform) [inline config/database.yml]: Please change the adapter name

File: config/database.yml [hunk 2]
```diff
@@ -8,18 +8,21 @@ default: &default
...
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
+  host: localhost
+  username: ENV['POLYMORPHIC_APP_DATABASE_USER']
+  password: ENV['POLYMORPHIC_APP_DATABASE_PASSWORD']
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: polymorphic-association_development
```
Comment (poonamsimform) [inline config/database.yml]: Please use ENV for database_name

File: .gitignore [hunk 3]
```diff
@@ -33,3 +33,4 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+/config/local_env.yml
```
Comment (poonamsimform) [inline .gitignore:36]: FYI: you can also use `dot_env` gem or `figaro` gem. Please learn about the same

Comment (ichiragrathod) [reply .gitignore:36]: Sure!

File: Gemfile [hunk 4]
```diff
@@ -12,7 +12,8 @@ gem 'rails', '~> 7.0.3'
 gem 'sprockets-rails'
 
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3', '~> 1.4'
+# gem 'sqlite3', '~> 1.4'
```
Comment (poonamsimform) [inline Gemfile]: Please remove this, no need to comment this gem. Will now onwards use the pg.

**Review by poonamsimform — APPROVED** [review]

**Pull request merged**